stdlib/influxdata/influxdb/schema/show_tag_values_measurement_or_predicate_test.flux

```diff
@@ -71,12 +71,12 @@ testcase show_tag_values_measurement_or_predicate {
 
     want = csv.from(
         csv: "#datatype,string,long,string
-#group,false,false,false
-#default,0,,
-,result,table,_value
-,,0,north
-,,0,south
-",
+    #group,false,false,false
```

Comment:
I assume the `safe_to_reindent` flag is supposed to prevent this indentation? Do these files need reformatting?

Reply:
oh boy, yeah there seems to still be a bug with csv. Lemme take a look.